models/src/main/thrift/events/crier/event.thrift

```diff
@@ -2,22 +2,47 @@ namespace scala com.gu.crier.model.event.v1
 
 include "content/v1.thrift"
 
+/*
+ * For large update events the EventType is set to RetrievableUpdate
```

Comment:
it's just a bit too long to be inlined, and I think we agreed of leaving a comment because it's not entirely obvious just by looking at the model why we're doing this (think people outside capi?)